src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -93,9 +119,13 @@ internal sealed class MeterProviderSdk : MeterProvider
                 shouldListenTo = instrument => meterSourcesToSubscribe.Contains(instrument.Meter.Name);
             }
 
+            OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent($"Listening to following meters = \"{string.Join(";", meterSources)}\".");
```

Comment:
make these non-events, else the cost is paid even if none is listening to these logs.